Casamari/MNC007.xml

```diff
@@ -128,136 +223,235 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                              <seg part="I">
+                                 <locus target="#2v"/>ሰላም፡ ለከ፡ ተክለ፡ ሃይማኖት፡ አቡነ፡ ምሉዓ፡ ሞገስ፡ ወጸጋ፡ ስፉሃ፡
+                                 አእፁቅ፡ ወወይን፡ እንበለ፡ ንትጋ፡ ለባሕር፡ እስከ፡ ድንጋጋ፡ ሰ፡ እም፡ ልደትከ፡ ሕፃን፡</seg>
+                           </q>
+                        </item>
+
+                        <item xml:id="e1">
+                           <desc> Scribal corrections occur rarely. Erroneous characters or words
+                              are encircled or marked with lines above and below (e.g. <locus
+                                 target="#34v #35r #37r"/>). Scribbles on <locus from="1r" to="2r"
+                              />.</desc>
+                        </item>
+
+                        <item xml:id="e2">
+                           <desc>A sticker of the monastic library with the signature ‘Ms 35’ is
+                              glued onto the inner side of the front board. The signature Et I is
+                              written in pencil in the lower margin of <locus target="#1r"/>. The
+                              inventory number 50432 is written in pencil on <locus target="#44v"
+                              />.</desc>
+                        </item>
+                     </list>
                   </additions>
-            
+
                   <bindingDesc>
-                     <binding contemporary="true">
-                        <decoNote xml:id="b1">Two wooden boards covered with leather. Three pairs of sewing stations. 
-                           European binding.
-                        </decoNote>
-                        <decoNote xml:id="b2" type="bindingMaterial">
+                     <!-- TO BE CHECKED BY MARCO-->
+                     <binding contemporary="true" xml:id="binding">
+                        <decoNote xml:id="b1">European binding. The original binding is lost. The
+                           manuscript was professionally rebound with new material.</decoNote>
+                        <decoNote xml:id="b2">Two<material key="wood">wooden</material> boards
+                           covered with reddish-brown <material key="leather"/>, parchment
+                           paste-downs.</decoNote>
+                        <decoNote xml:id="b3" type="bindingMaterial">
                            <material key="wood"/>
+                           <material key="leather"/>
+                           <material key="parchment"/>
+                        </decoNote>
+                        <decoNote xml:id="b4" type="Cover" color="reddish-brown">Quarter cover with
+                           the library's signature sticker printed on the spine; below the shelf
+                           mark ‘Ms. ET. I’ tooled with golden colour.</decoNote>
+                        <decoNote xml:id="b5" type="SewingStations">6 (The original binding showed
+                           only four sewing stations. The Ms received additional two sewing stations
```

Comment:
"two additional"
This is a common situation in many of the restored Casamari MSS. I think we can decide (together with Marco?) a consistent way to deal with this issue for all the involved MSS, and decide what to encode under Condition and what under SewingStations. In alternative to the proposed solution we can for example be as much concise as possible in `<decoNote xml:id="b5" type="SewingStations">` just giving the current number of stations (6), while we provide the history of the binding in conservation.

Reply:
Yes, you are right. We could discuss it at our next meeting or wait for Marco's descriptions.
I noted it down.